superdesk/media/media_editor.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013-2018 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+from superdesk.resource import Resource, not_analyzed
+from superdesk.services import BaseService
+from superdesk.media.renditions import generate_renditions, get_renditions_spec
+from superdesk import get_resource_service
+from superdesk import errors
+from flask import request, current_app
+from PIL import Image, ImageEnhance
+from io import BytesIO
+import os.path
+import uuid
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MediaEditorResource(Resource):
+    schema = {
+        'item_id': {
+            'type': 'string',
+            'mapping': not_analyzed
+        },
+        'item': {
+            'type': 'dict',
+            'mapping': not_analyzed
+        },
+        'edit': {
+            'type': 'dict',
+            'required': True,
+            'mapping': not_analyzed
+        }
+    }
+    internal_resource = False
+    resource_methods = ['GET']
```

Comment:
why is it actually using `GET` method? imo it should be `POST` - imo it changes state

Reply:
It doesn't store items in database, files are temporary, I though `GET` was more appropriate in this specific case.

Reply:
I think we also use post with picture crop, so it would be more consistent
https://github.com/superdesk/superdesk-core/blob/master/apps/picture_crop/__init__.py#L50

Reply:
alright, I'll change it.

Reply:
@petrjasek done